code/game/objects/objs.dm

```diff
@@ -194,7 +194,7 @@
 
 /obj/proc/tesla_act(var/power)
 	being_shocked = 1
-	var/power_bounced = power / 2
+	var/power_bounced = power / 1.3

```

Comment:
@Fox-McCloud you'd be better off asking @RemieRichards; she'd know better than I